subgraphs/exchange/config/goerli.json

```diff
@@ -30,5 +30,13 @@
             "address": "0xfD474E4809e690626C67ECb7A908de4b9c464b99",
             "startBlock": "4851367"
         }
-    ]
-}
\ No newline at end of file
+  ],
+    "LIMITORDERV1": [
+        {
+            "name": "LimitOrder",
+            "network": "goerli",
+            "address": "0x7B955545428B3a0E7520EEfF0DC30D740F290e4b",
+            "startBlock": "7497880"
+        }
+  ]
```

Comment:
nit: The padding seems not consistent

Reply:
Same for the mainnet

Reply:
Yes, this will be fixed in the next commit!